packages-next/admin-ui/src/system/generateAdminUI.ts

```diff
@@ -76,10 +76,12 @@ export const generateAdminUI = async (config: KeystoneConfig, system: KeystoneSy
   // Add files to pages/ which point to any files which exist in admin/pages
   const userPagesDir = Path.join(process.cwd(), 'admin', 'pages');
   const files = await fastGlob('**/*.{js,jsx,ts,tsx}', { cwd: userPagesDir });
+  const makeImportPath = (path: string) => path.replace(new RegExp('\\' + Path.sep, 'g'), '/');
+
   await Promise.all(
     files.map(async filename => {
       const outputFilename = Path.join(projectAdminPath, 'pages', filename);
-      const path = Path.relative(Path.dirname(outputFilename), Path.join(userPagesDir, filename));
+      const path = makeImportPath(Path.relative(Path.dirname(outputFilename), Path.join(userPagesDir, filename)));
       await fs.outputFile(outputFilename, `export { default } from "${path}"`);
```

Comment:
```suggestion
      const path = Path.relative(Path.dirname(outputFilename), Path.join(userPagesDir, filename));
      // Convert filesystem path separator to the `/` expected in JS imports
      const importPath = path.replace(new RegExp(Path.sep, 'g'), '/');
      await fs.outputFile(outputFilename, `export { default } from "${importPath}"`);
```
Rather than using a function here it would be clearer to have inline code with a comment.